test/unit/path.js

```diff
@@ -127,11 +127,6 @@
     updatePath(path, REFERENCE_PATH_OBJECT.path, true);
     assert.deepEqual(path.toObject(), REFERENCE_PATH_OBJECT);
     updatePath(path, REFERENCE_PATH_OBJECT.path, false);
-    var left = path.left;
-    var top = path.top;
-    path.center();
-    assert.equal(left, path.left);
-    assert.equal(top, path.top);
```

Comment:
the path didn't have a canvas, so this call to path.center() was doing nothing.

Reply:
great seeing junk removed
we cleaned up a lot
and more will come for sure